yaptide/routes/common_sim_routes.py

```diff
@@ -109,6 +108,11 @@ def post():
 def get_single_estimator(sim_id: int, estimator_name: str):
     """Retrieve a single estimator by simulation ID and estimator name"""
     estimator = fetch_estimator_by_sim_id_and_est_name(sim_id=sim_id, est_name=estimator_name)
+
+    if not estimator:
```

Comment:
The aim of the PR is to introduce clear distinction between name and file_name.
It's very confusing to have a method which expects `estimator_name` as argument and is using it as it would be a filename. 

As the user of this code I would expect two methods: `get_single_estimator_by_name`  and `get_single_estimator_by_filename`